### PR TITLE
Hubspot: page location table [INTEG-2780]

### PR DESCRIPTION
**Pull request by FBanfi**
## Purpose

This update includes the table for the page location connected entries.

## Approach

The table was created and some edge cases were handled (such as "loading state", "no connected entries" and "error when fetching").

## Testing steps

Automated test were added.

https://github.com/user-attachments/assets/f81dce4c-0ab6-47c0-b201-0b4922809100

Error message:
Captura de pantalla 2025-07-15 a la(s) 9 46 02 a  m

## Breaking Changes

N/A

## Dependencies and/or References

Link to [INTEG-2780](https://contentful.atlassian.net/jira/software/c/projects/INTEG/boards/3109?issueParent=367108&label=10-Pines&selectedIssue=INTEG-2780) ticket.

## Deployment

N/A

[INTEG-2780]: https://contentful.atlassian.net/browse/INTEG-2780?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/hubspot/src/locations/Page.styles.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    width: '6rem',
+  }),
+  loading: css({
+    height: '80vh',
+  }),
+  modalMainContainer: css({
+    minWidth: 400,
+    maxHeight: '50vh',
+  }),
+  modalEntryContainer: css({
```
Comment (JuliRossi) [inline apps/hubspot/src/locations/Page.styles.ts]: 💭 Are we using these styles that reference the modal? We can leave it if you think we'll use it on the actual modal in the future!

File: apps/hubspot/src/locations/Page.tsx [hunk 2]
```diff
@@ -1,16 +1,249 @@
...
+            locale={locale}
+            onManageFields={handleManageFields}
+          />
+        )}
+      </Box>
+    </Flex>
+  );
+};
+
+const ConnectedEntriesTable = ({
```
Comment (JuliRossi) [inline apps/hubspot/src/locations/Page.tsx]: 💭 We could create a separate component for this! Since it seems to be similar to braze . We can leave this for the next page view PR

File: apps/hubspot/src/locations/Page.tsx [hunk 3]
```diff
@@ -1,16 +1,249 @@
...
+  entries,
+  connectedFields,
+  displayFieldId,
+  locale,
+  onManageFields,
+}: {
+  entries: any[];
```
Comment (JuliRossi) [inline apps/hubspot/src/locations/Page.tsx]: 💭 This shouldn't be any

File: apps/hubspot/src/locations/Page.tsx [hunk 4]
```diff
@@ -1,16 +1,249 @@
...
-  const sdk = useSDK<PageAppSDK>();
-  /*
-     To use the cma, inject it as follows.
-     If it is not needed, you can remove the next line.
-  */
-  // const cma = useCMA();
-
-  return <Paragraph>Hello Page Component (AppId: {sdk.ids.app})</Paragraph>;
+  const sdk = useSDK();
+  const [entries, setEntries] = useState<any[]>([]);
```
Comment (JuliRossi) [inline apps/hubspot/src/locations/Page.tsx]: 💭 This shouldn't be any
